BankSDK/GiniBankSDKExample/GiniBankSDKExample/ComponentAPI/ComponentAPICoordinator.swift

```diff
@@ -472,7 +472,7 @@ extension ComponentAPICoordinator: UINavigationControllerDelegate {
         if toVC is CameraViewController &&
             (fromVC is ReviewViewController ||
                 fromVC is AnalysisViewController ||
-                fromVC is ImageAnalysisNoResultsViewController) {
+                fromVC is HelpTipsViewController) {
             // When going directly from the analysis or from the single page review screen to the camera the pages
```

Comment:
fromVC is ImageAnalysisNoResultsViewController) {

we need to use ImageAnalysisNoResultsViewController as well

Reply:
ok, I missed it.